mesh/mesh.cpp

```diff
@@ -1581,6 +1581,116 @@ void Mesh::SetAttributes()
    }
 }
 
+void Mesh::GetAttributeSetNames(std::set<std::string> &names) const
+{
+   names.clear();
+
+   for (auto const &attr_set : attr_sets)
+   {
+      names.insert(attr_set.first);
+   }
+}
+
+void Mesh::GetBdrAttributeSetNames(std::set<std::string> &names) const
+{
+   names.clear();
+
+   for (auto const &bdr_attr_set : bdr_attr_sets)
+   {
+      names.insert(bdr_attr_set.first);
+   }
+}
+
+
+void Mesh::SetAttributeSet(const std::string &set_name, const Array<int> &attr)
+{
+   attr_sets[set_name] = attr;
```

Comment:
What is the motivation that `attr` is not sorted and unified here ?

Reply:
No motivation. Just an oversight. Good catch!